code/modules/paperwork/desk_bell.dm

```diff
@@ -63,15 +57,29 @@
 			return TRUE
 		return FALSE
 
-// Deconstruct
+// Deconstruct and Anchor
 /obj/item/desk_bell/wrench_act(mob/living/user, obj/item/tool)
-	to_chat(user, "<span class='notice'>You begin taking apart the bell...</span>")
-	if(tool.use_tool(src, user, 5 SECONDS))
-		to_chat(user, "<span class='notice'>You disassemble the bell...</span>")
-		playsound(user, 'sound/items/deconstruct.ogg', 50, vary = TRUE)
-		new /obj/item/stack/sheet/metal(drop_location(), 2)
-		qdel(src)
-		return TRUE
+	. = TRUE
+	if(user.a_intent == INTENT_HARM && !in_inventory)
+		to_chat(user, "<span class='notice'>You begin taking apart the bell...</span>")
+		if(tool.use_tool(src, user, 5 SECONDS))
+			to_chat(user, "<span class='notice'>You disassemble the bell...</span>")
```

Comment:
Probably don't want ellipses on the completion message.
```suggestion
			to_chat(user, "<span class='notice'>You disassemble the bell.</span>")
```

Reply:
Might also want to add some completion messages for the other tool actions while you're at it